recipe/meta.yaml

```diff
@@ -9,7 +9,7 @@ source:
   sha256: c66d13844ed5d1f157829e90c1d4b3fd92d4292beaa6cfc3b610d541e0ddb881
 
 build:
-  number: 0
+  number: 1
```

Comment:
I was going back and forth about whether I need to bump the build number.  Best to just do it, I think.